src/arkit/ARKitDevice.js

```diff
@@ -112,7 +123,9 @@ export default class ARKitDevice extends PolyfilledXRDevice {
 		})
 
 		let watchResult = await this._arKitWrapper.watch(ARKitOptions).then((results) => {
-			const session = new Session(options.outputContext || null)
+			// Note: Commenting out options.outputContext for now because
+			//       I don't know what it's used for.
+			const session = new Session(/*options.outputContext ||*/ null)
```

Comment:
Any comments about this `.outputContext` comment out?

Reply:
in requestSession, we used to have to pass in outputContext as the webgl context.  Do we do that anymore?

Reply:
I think we no longer pass webgl context via `navigator.xr.requestSession()`.

https://immersive-web.github.io/webxr/#xr-interface
https://immersive-web.github.io/webxr/#feature-dependencies

Reply:
makes sense.  Ya, just ignore it then.

Reply:
Cleaned up, thanks